spec/integration/admin_sees_individual_orders_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.feature "admin sees orders" do
+  scenario "admin can see an individual orders" do
+    items = create_list(:item, 2)
+    admin = create(:user, role: 1)
+    user = create(:user)
+    order = user.orders.create
+    cart = Cart.new({ items[0].id => 1, items[1].id => 1 })
+    order.create_order_items(cart)
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+    visit order_path(order)
+
+    within("h4") do
```

Comment:
I ran into some issues previously when fixing tests for views with 'within' calls using just html tags - primarily ambiguous results. Could we try just specifying this as a specific id instead?